utils.py

```diff
@@ -179,6 +182,40 @@ def send_flow_mods_event(
         )
 
 
+def send_flow_mods_http(
```

Comment:
Highly recommend that you decorate it with tenacity's `@retry` similar pattern that we have [on other NApps](https://github.com/kytos-ng/telemetry_int/blob/master/kytos_api_helper.py#L22), if the http server is temporarily overloaded this can help to succeed in certain transient cases

Reply:
Tenacity has been added to the endpoint.